src/System.Runtime/src/ApiCompatBaseline.uap101aot.txt

```diff
@@ -431,4 +431,16 @@ MembersMustExist : Member 'System.Threading.Tasks.Task.Dispose(System.Boolean)'
 CannotRemoveBaseTypeOrInterface : Type 'System.Threading.Tasks.Task<TResult>' does not implement interface 'System.IDisposable' in the implementation but it does in the contract.
 MembersMustExist : Member 'System.Attribute.IsDefaultAttribute()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Attribute.Match(System.Object)' does not exist in the implementation but it does exist in the contract.
-MembersMustExist : Member 'System.Attribute.TypeId.get()' does not exist in the implementation but it does exist in the contract.
\ No newline at end of file
+MembersMustExist : Member 'System.Attribute.TypeId.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
I think that once we remove one of the properties for typeId, there should only be one of them here.